scripts/submit_dpo_job.py

```diff
@@ -20,7 +20,7 @@ def main():
                         help="Path to the default Beaker config file")
     parser.add_argument("--config", default=None,
                         help="Path to an additional config file to override default settings")
-    parser.add_argument("--cluster", type=str, default="ai2/allennlp-cirrascale", help="Beaker cluster to use")
+    parser.add_argument("--cluster", type=str, default="ai2/allennlp", help="Beaker cluster to use")
```

Comment:
maybe the default should be jupiter here.

Reply:
DOne.